scripts/build/setup/termux_setup_protobuf.sh

```diff
@@ -14,7 +14,7 @@ termux_setup_protobuf() {
 			termux_download \
 				https://github.com/protocolbuffers/protobuf/releases/download/v$_PROTOBUF_VERSION/$_PROTOBUF_ZIP \
 				$TERMUX_PKG_TMPDIR/$_PROTOBUF_ZIP \
-				a2900100ef9cda17d9c0bbf6a3c3592e809f9842f2d9f0d50e3fba7f3fc864f0
+				544667cf6b55cac1e90e582f5bf2651b70de0e36e79dc3ac69ca80fb053dcc01
```

Comment:
This hash is not correct (for protobuf 3.14)

Reply:
Sorry my mistake, i calculated the checksum without following redirect, i'll remove this.